--- old
+++ new
@@ -9,1 +9,1 @@
-Sau bữa tối thân mật ăn uống rượu bét nhè, t với thằng thủ trưởng đi dạo. Tối miền quê cũng không có gì, yên tĩnh thanh bình. Chúng tôi đi dạo qua nhà thờ rồi tạt vào tiệm tạp hóa mua kem. Kem que có loại giá 2k, có loại 5k, chúng tôi lấy luôn chục cây thay vì 2 cây như dự tính ban đầu. 
+Sau bữa tối thân mật ăn uống rượu bét nhè, t với thằng thủ trưởng đi dạo. Tối miền quê cũng không có gì, yên tĩnh thanh bình. Chúng tôi đi dạo qua nhà thờ rồi tạt vào tiệm tạp hóa mua kem. Kem que có loại giá 2k, có loại 5k, chúng tôi lấy luôn chục cây thay vì 2 cây như dự tính ban đầu.  