--- old
+++ new
@@ -6,1 +6,3 @@
-Khác với dịp 30/4 - 1/5 mọi năm, năm nay t xuống Kiên Giang bê quả cho Thắng. Cũng nhờ vậy mà t có ý tưởng để làm bài nói 2, không phải bịa hoàn toàn. Từ chỗ nhà cô dâu đến bờ kè tp Rạch Giá khoảng 40km nên t k có dịp tham quan rồi. 
+Khác với dịp 30/4 - 1/5 mọi năm, năm nay t xuống Kiên Giang bê quả cho Thắng. Cũng nhờ vậy mà t có ý tưởng để làm bài nói 2, không phải bịa hoàn toàn. 
+Chúng tôi tập trung tại nhà người quen của Thắng tại bến Vân Đồn
+Từ chỗ nhà cô dâu đến bờ kè tp Rạch Giá khoảng 40km nên t k có dịp tham quan rồi. 