--- old
+++ new
@@ -6,1 +6,1 @@
-Cầu Ba Son có trụ tháp nghiêng và những dây văng có đèn chiếu sáng thay đổi màu sắc trông thật đẹp, lung linh vào ban đêm. Từ ga metro Ba Son, t đi lên những bậc thang rồi đi bộ trên cầu Ba Sonm
+Từ ga metro Ba Son, t đi lên những bậc thang rồi đi bộ trên cầu Ba Son. Cầu có trụ tháp nghiêng và những dây văng có đèn chiếu sáng thay đổi màu sắc trông thật đẹp, lung linh vào ban đêm. Đi bộ khoảng 15 thì tới quán Om Ba Sơn, mọi người đã có mặt từ trước đó rồi. 